common/djangoapps/util/model_utils.py

```diff
@@ -152,6 +155,10 @@ def emit_setting_changed_event(user, db_table, setting_name, old_value, new_valu
         truncated_fields
     )
 
+    # Announce field change
+    USER_FIELD_CHANGED.send(sender=None, user=user, table=db_table, setting=setting_name,
```

Comment:
If you look at https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/models.py#L71, you can actually see that `emit_setting_changed_event` is being called from inside a receiver: `@receiver(post_save, sender=UserPreference)`. Rather than adding a new signal, it would be better for the email marketing code to use the existing signal.